src/vivarium/component.py

```diff
@@ -92,34 +92,41 @@ def initialization_columns_required(self) -> List[str]:
         """
         return []
 
+    @property
+    def post_setup_priority(self) -> int:
+        """The priority of this component's post_setup listener if it exists."""
+        return 5
```

Comment:
Should we continue using bald integers here? Like would it be better to have this defined as a constant elsewhere?

Reply:
Good idea